nanomq_cli/include/main.h

```diff
@@ -9,5 +9,6 @@
 #include "vsomeip_gateway.h"
 #include "rule_ctrl.h"
 #include "dds_client.h"
+#include "nftp _client.h"
```

Comment:
An typo????

Reply:
what's wrong with that? That's not a typo.

Reply:
That is a typo, my bad.